### PR TITLE
The 3rd times the charm pull request..

### PR DESCRIPTION
**Pull request by DivaVocals**
Clean up <script> tags to use proper HTML..

Re: files missing line breaks: The missing line breaks are inherited from the Zen Cart master repo. I have made the changes, but they are NOT part of this PR as of it's original submission. I will GLADLY  will submit them as an update to this pull request or as a separate pull request if needed/wanted.

### CODE REVIEW

File: admin/packingslip.php [hunk 1]
```diff
@@ -38,7 +38,7 @@
 <meta http-equiv="Content-Type" content="text/html; charset=<?php echo CHARSET; ?>">
 <title><?php echo TITLE; ?></title>
 <link rel="stylesheet" type="text/css" href="includes/stylesheet.css">
-<script language="javascript" src="includes/menu.js"></script>
+<script type="text/javascript"src="includes/menu.js"></script>
```
Comment (drbyte) [inline admin/packingslip.php]: Somehow you dropped a space before `src` here.

File: includes/classes/payment.php [hunk 2]
```diff
@@ -160,7 +160,7 @@ function javascript_validation() {
          $js .= '   return collectsCardDataOnsite(payment_value);' . "\n";
        }
        $js .= '    return true;' . "\n";
-       $js .= '  }' . "\n" . '}' . "\n" . '//--></script>' . "\n";
+       $js .= '  }' . "\n" . '}' . "\n" . '</script>' . "\n";
```
Comment (drbyte) [inline includes/classes/payment.php]: Missed the matching opening comment up on line 123

File: includes/form_check.js.php [hunk 3]
```diff
@@ -175,4 +175,4 @@ function check_form(form_name) {
     return true;
   }
 }
-//--></script>
```
Comment (drbyte) [inline includes/form_check.js.php]: missed the matching opening comment up on line 12

File: includes/modules/additional_images.php [hunk 4]
```diff
@@ -108,7 +108,7 @@
 
 
     // Link Preparation:
-    $script_link = '<script><!--' . "\n" . 'document.write(\'' . ($flag_display_large ? '<a href="' . $js_href . '"' . ($js_rel != '' ? ' rel="' . $js_rel . '"' : '') . ' title="' . addslashes($products_name) . '">' . $thumb_slashes . '<br />' . TEXT_CLICK_TO_ENLARGE . '</a>' : $thumb_slashes) . '\');' . "\n" . '//--></script>';
+    $script_link = '<script><!--' . "\n" . 'document.write(\'' . ($flag_display_large ? '<a href="' . $js_href . '"' . ($js_rel != '' ? ' rel="' . $js_rel . '"' : '') . ' title="' . addslashes($products_name) . '">' . $thumb_slashes . '<br />' . TEXT_CLICK_TO_ENLARGE . '</a>' : $thumb_slashes) . '\');' . "\n" . '</script>';
```
Comment (drbyte) [inline includes/modules/additional_images.php]: missed removing the opening comment at the beginning of the line

**Comment by drbyte** [comment]
Lots of opening `<!--` not removed but the closing `//-->` is, looks like mostly all catalog-side.

**Comment by DivaVocals** [comment]
So the 3rd times NOT the charm eh?? :laughing: Okay.. let me look at this later today..

File: includes/classes/payment.php (hunk 2 above)
Comment (DivaVocals) [inline includes/classes/payment.php]: Fixed

File: includes/form_check.js.php (hunk 3 above)
Comment (DivaVocals) [inline includes/form_check.js.php]: Fixed..

**Comment by drbyte** [comment]
After a quick visual review I think that's looking good. 

Will run it through some regression testing.

**Comment by DivaVocals** [comment]
Cool let me know.. :smile: Again, thanks for your help.. Rocky start, but I will do better the next time..

File: admin/invoice.php [hunk 5]
```diff
@@ -35,12 +35,12 @@
 <meta http-equiv="Content-Type" content="text/html; charset=<?php echo CHARSET; ?>">
 <title><?php echo TITLE; ?></title>
 <link rel="stylesheet" type="text/css" href="includes/stylesheet.css">
-<script language="javascript" src="includes/menu.js"></script>
-<script language="javascript" type="text/javascript"><!--
+<script type="text/javascript"src="includes/menu.js"></script>
```
Comment (drbyte) [inline admin/invoice.php]: missing a space before `src`

Comment (DivaVocals) [inline admin/invoice.php]: Fixed..

**Comment by drbyte** [comment]
:+1:

**Comment by DivaVocals** [comment]
Do I get an A now teacher??? :lollipop:

**Comment by ajeh** [comment]
:+1:

**Comment by DivaVocals** [comment]
:+1: Pardon me if this sounds silly.. but I think I am actually gonna do the happy dance over here!!! :laughing:

**Comment by drbyte** [comment]
You'd be perfectly entitled to do that!

It's great to have community members contributing. We appreciate it!

**Comment by DivaVocals** [comment]
More to come.. :+1:

**Comment by DivaVocals** [comment]
Okay.. so I'm CRYSTAL clear on process.. I would now do a PR from the main v1.6.0 branch to update my forks v1.60 branch, then create a NEW branch in my fork based on the updated v1.6.0 branch from which I can submit any new PRs.. Correct???

**Comment by drbyte** [comment]
For keeping my dev branch up to date I don't think I've ever tried doing a PR in reverse. 
The Zen Cart dev team uses a "multiple remotes" approach, as outlined here: http://docs.zen-cart.com/Contributing/master/github_workflow <http://docs.zen-cart.com/Contributing/master/github_workflow>

But, yes, you need to update your fork to the latest from the zc v160 branch, then make a new feature branch based on that updated v160 branch.

**Comment by drbyte** [comment]
Link updated to http://docs.zen-cart.com/Contributing/main/github_workflow

**Comment by DivaVocals** [comment]
Okay.. I'll take a looksee at that link.. thanks again.. I think I'm having fun now.. :smile_cat:

**Pull request merged**